test/envoy/core_test.clj

```diff
@@ -19,9 +19,33 @@
 (defenv :envoy-bad-type
   "Re-declaring a variable.")
 
+(defenv :envoy-custom-parser
+  "Specifying a special parsing fn."
+  :parser clojure.edn/read-string)
+
+(ns envoy.core-test.special)
```

Comment:
:-1: on changing the namespace in the middle here. Why do you want to do this in another namespace? Isn't `clojure.string/lower-case` _already_ an example of a function in another ns?

Reply:
> Isn't clojure.string/lower-case already an example of a function in another ns?
Well, _yes_ I suppose :)

Removed.